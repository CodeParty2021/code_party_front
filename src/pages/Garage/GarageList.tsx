import React from "react";
import { Link } from "react-router-dom";

type Prop = {};

export const GarageList: React.FC<Prop> = () => {
  return (
    <div>
      <h1>ガレージ画面</h1>
      <h1>ロボットの新規作成</h1>
      <h1>ここにロボットが並んで選択できる</h1>
      <Link to="/garage/:id">ロボットの詳細を表示する(リンクの例)</Link>
    </div>
  );
};