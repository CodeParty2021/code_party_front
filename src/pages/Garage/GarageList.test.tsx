import React from "react";
import { shallow } from "enzyme";
import { GarageList } from "./GarageList";
describe("<RobotDevelopmentTop />", () => {
  it("auth snapshot test", () => {
    const wrapper = shallow(< GarageList/>);

    expect(wrapper.getElements()).toMatchSnapshot();
  });
});