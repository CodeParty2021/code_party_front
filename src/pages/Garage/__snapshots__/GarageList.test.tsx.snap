// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RobotDevelopmentTop /> auth snapshot test 1`] = `
Array [
  <div>
    <h1>
      ガレージ画面
    </h1>
    <div>
      ロボットの新規作成画面 ここにロボットが並んで選択できる
    </div>
    <Link
      to="/garage/:id"
    >
      ロボットの詳細を表示する(リンクの例)
    </Link>
  </div>,
]
`;
